ibm_i/assets/configuration/spec.yaml

```
@@ -94,4 +94,21 @@ files:
           - name: "job_queue"
           - name: "message_queue_info"
         default: *queries_default
+    - name: message_queue_info
+      description: |
+        A configuration mapping for the message_queue_info query.
+
+        selected_message_queues filters the message queues to make your queries less computer-intensive.
```

I would rewrite it with something like this:
```suggestion
        selected_message_queues restricts the message_queue_info query to a subset of the queues. This can reduce the CPU usage on the target IBM i machine.
```

---

You're right ! Done in [b490ac1](https://github.com/DataDog/integrations-core/pull/12808/commits/b490ac14f5259b46bc732e16e5ce1cfdde3c6bae).